### ISSUE TITLE
`sensitive` and `hide-empty` settings are not working as expected

### ISSUE THREAD

**Issue by ChernikovP** [terraform-docs/terraform-docs#568]
### Describe the bug

Based on my understanding of https://terraform-docs.io/user-guide/configuration/settings/:

- `hide-empty: true` should result in not printing something like
  ```
  ## Outputs

  No outputs.
  ```
 - `sensitive: true` should add corresponding column to `Inputs` table.
 
 None of this holds true currently.

### How can we reproduce it?

variables.tf:
```hcl
variable "foo" {
  description = "foo var desc"
  type        = string
}
variable "bar" {
  description = "bar var desc"
  type        = string
  default     = "bar"
  sensitive   = true
}
```

main.tf:
```hcl
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "3.60.0"
    }
  }
  required_version = "1.0.7"
}
provider "aws" {
  region = "us-west-1"
}
data "aws_region" "current" {}
```
.terraform-docs.yaml:
```yaml
formatter: "markdown table"
version: ">= 0.15.0"
sections:
  show:
    - inputs
    - outputs
settings:
  hide-empty: true
  sensitive: true
```
Running `terraform-docs .` results in:
```
## Inputs

| Name | Description | Type | Default | Required |
|------|-------------|------|---------|:--------:|
| <a name="input_bar"></a> [bar](#input\_bar) | bar var desc | `string` | `"bar"` | no |
| <a name="input_foo"></a> [foo](#input\_foo) | foo var desc | `string` | n/a | yes |

## Outputs

No outputs.
```

### Environment information

- terraform-docs version (use `terraform-docs --version`):
  ```
  pchernikov@pchernikov-mbp dummy-example % terraform-docs version
  terraform-docs version v0.15.0 darwin/amd64
  ```
- OS  - MacOS Catalina 10.15.5

**Comment by pascal-hofmann** [comment]
I think these features are unreleased yet.

**Comment by ChernikovP** [comment]
@pascal-hofmann , it looks like you're right with regards to `hide-empty`:

>  ### hide-empty
>  > since: `v0.16.0`
>  > scope: `asciidoc`, `markdown`
>
>  Hide empty sections.

But as for `sensitive` feature, it should be out since `0.10.0`:

>  ### sensitive
>  > since: `v0.10.0`
>  > scope: `asciidoc`, `markdown`
>
>  Show “Sensitive” as column (in table format) or section (in document format).

**Comment by khos2ow** [comment]
@ChernikovP as you found, `hide-empty` is gonna be available in the upcoming `v0.16.0` release. But `sensitive` config, as it stands now, only works for outputs, that needs to be implemented and will be available in the following release after `v0.16.0`.

**Comment by khos2ow** [comment]
I'm gonna close this in favor of #567, which is dedicated to sensitive for inputs.
